DigitalLearningSolutions.Data/DataServices/UserDataService/AdminUserDataService.cs

```diff
@@ -51,7 +51,7 @@ ELSE cc.CategoryName
             return user;
         }
 
-        public List<AdminUser> GetAdminUsersByUsername(string username)
+        public AdminUser? GetAdminUserByUsername(string username)

```

Comment:
I think this method deserves an XML doc to explain that it will throw an InvalidOperationException if multiple admin users are found, and explaining that that will only happen if one admin's login matches another's email address, which is very unlikely.